docs/concepts/fields.md

```diff
@@ -831,7 +831,38 @@ The property will also be taken into account in the JSON Schema.
     However, note that Pydantic will *not* perform any additional logic on the wrapped property
     (validation, cache invalidation, etc.).
 
-Here's an example:
+Here's an example of the JSON schema (in serialization mode) generated for a model with a computed field:
+
+```py
+from pydantic import BaseModel, computed_field
+
+
+class Box(BaseModel):
+    width: float
+    height: float
+    depth: float
+
+    @computed_field
+    @property  # (1)!
+    def volume(self) -> float:
+        return self.width * self.height * self.depth
+
+
+print(Box.model_json_schema(mode='serialization'))
+#>{
+#>    'properties': {
+#>        'width': {'title': 'Width', 'type': 'number'},
+#>        'height': {'title': 'Height', 'type': 'number'},
+#>        'depth': {'title': 'Depth', 'type': 'number'},
+#>        'volume': {'readOnly': True, 'title': 'Volume', 'type': 'number'},
+#>    },
+#>    'required': ['width', 'height', 'depth', 'volume'],
+#>    'title': 'Box',
+#>    'type': 'object',
+#>}
```

Comment:
You'll want to use the `"""` blocks here, not `#>`. That should fix the formatting.

Reply:
Thank you! Have just changed it so hopefully the checks pass now